src/main/java/de/flapdoodle/embed/mongo/config/DownloadConfigBuilder.java

```diff
@@ -44,12 +58,22 @@ public DownloadConfigBuilder defaults() {
 		fileNaming().setDefault(new UUIDTempNaming());
 		downloadPath().setDefault(new PlattformDependendDownloadPath());
 		progressListener().setDefault(new StandardConsoleProgressListener());
-		artifactStorePath().setDefault(new UserHome(".embedmongo"));
+		artifactStorePath().setDefault(defaultArtifactDownloadLocation());
 		downloadPrefix().setDefault(new DownloadPrefix("embedmongo-download"));
 		userAgent().setDefault(new UserAgent("Mozilla/5.0 (compatible; Embedded MongoDB; +https://github.com/flapdoodle-oss/embedmongo.flapdoodle.de)"));
 		return this;
 	}
 
+	private IDirectory defaultArtifactDownloadLocation() {
+		String artifactDownloadLocationEnvironmentVariable = environmentVariables.get("EMBEDDED_MONGO_ARTIFACTS");
```

Comment:
Great, the pom was targeting 1.6 so I avoided Optional—which fed into my use of all the env vars rather than storing something that was potentially null. Tests and prod code are much nicer using Optional.

I’ve updated the source and target versions in the pom to 1.8 to allow me to use Optional.